addons/point_of_sale/static/src/js/Popups/AlertPopup.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('point_of_sale.AlertPopup', function(require) {
+    'use strict';
+
+    const AbstractAwaitablePopup = require('point_of_sale.AbstractAwaitablePopup');
+    const Registries = require('point_of_sale.Registries');
+    const { _lt } = require('@web/core/l10n/translation');
+
+    class AlertPopup extends AbstractAwaitablePopup {}
+
+    AlertPopup.template = 'AlertPopup';
+    AlertPopup.defaultProps = {
+        confirmText: _lt('Ok'),
+        title: '',
+        body: '',
+        cancelKey: false,
+    };
+
+    Registries.Component.add(AlertPopup);
+
+    return AlertPopup;
+});
```

Comment:
We are slowly but surely trying to adapt the code to how it should be defined without having to use the whole `odoo.define......`, `require`, etc and instead use proper import and export.
Can you take a look to `PromoCodeButton.js` to see how to adapt your file ? Don't hesitate to ask question .
Also, at the beginning of the file, it'll be better to put an alias like this
`/** @odoo-module alias=point_of_sale.AlertPopup**/